supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -359,6 +370,21 @@ Always spell out the full product name of the host, and do not capitalize the te
 
 *See also*: xref:syndesis[Syndesis], xref:fuse-online[Fuse Online]
 
+[discrete]
+[[red-hat-java]]
+==== image:images/no.png[no] Red Hat Java (noun)
+*Description*: Do not use _Red Hat Java_ to refer to the Red Hat distribution of the Open Java Development Kit (OpenJDK).
+
+Java is a registered trademark of Oracle and/or its affiliates.
```

Comment:
```suggestion
Java is a registered trademark of Oracle and its affiliates.
```
ISG says to avoid using "and/or". I think "and" covers the meaning here.

Reply:
Removed "/or". Thanks